community/cypher/docs/cypher-docs/src/docs/dev/ql/delete/index.asciidoc

```diff
@@ -2,9 +2,11 @@
 = Delete
```

Comment:
Added to the refcard. @nawroth - should DETACH DELETE show up in the TOC or be something under DELETE?